builds/install/misc/firebird.conf.in

```diff
@@ -768,6 +768,14 @@
 #
 #TcpNoNagle = 1
 
+#
+# Either enables or disables the "TCP Loopback Fast Path" feature (SIO_LOOPBACK_FAST_PATH).
+# Applies to Windows (version 8/2012 or higher) only.
+#
+# Type: Boolean, default 1 (true)
+#
+#TcpLoopbackFastPathOption = 1
+
```

Comment:
May I suggest naming it "TcpLoopbackFastPath", without the "Option" suffix. All config settings are options, wording looks redundant.

Reply:
you're right, I will change that

Reply:
done with Commit a0692f4